add-on/src/popup/quick-upload.js

```diff
@@ -134,6 +134,8 @@ function file2buffer (file) {
 
 function formatImportDirectory (path) {
   path = path.replace(/\/$|$/, '/')
+  path = path.replace(/(\/)\/+/g, '$1')
```

Comment:
I added this just because it makes a nicer user experience. Files will appear in the Web UI with multiple slashes, despite being stored without them if multiple slashes are not removed on format.

Reply:
Thanks! Do you mind adding unit tests for `formatImportDirectory`, so we don't break this during future refactors?

For now it can be `test/functional/lib/quick-upload.test.js` (similar to existing `test/functional/lib/ipfs-path.test.js`)